packaging/debian/debian/openvstorage-automation-lib.install

```diff
@@ -1,3 +1,4 @@
+ci          opt/OpenvStorage/ci
```

Comment:
I think there is a `/` required at the end, not sure though, but better safe than sorry